src/data/equiv/algebra.lean

```diff
@@ -328,3 +327,73 @@ instance symm.is_ring_hom {e : α ≃r β} : is_ring_hom e.to_equiv.symm := hom
   e.symm.to_equiv x = e.to_equiv.symm x := rfl
 
 end ring_equiv
+
+-- We first define the type of automorphisms on groups, additive monoids,
+-- additive groups and rings using `mul_equiv`, `add_equiv` and `ring_equiv`.
+-- In each case this type is also a group.
+
+namespace group
+
+def aut (γ : Type) [group γ] := mul_equiv γ γ
+
+@[extensionality] lemma ext_mul_equiv (α β : Type) [group α] [group β] (f : mul_equiv α β)
```

Comment:
Write `(f g : mul_equiv α β)` when you have multiple arguments of the same type.

Reply:
Do I change these and then commit again?

Reply:
Yes. If you just push more commits to the same branch, the pull request will automatically be updated.

You should then come back here and respond to the comment (simply with "Done" or "Fixed" is fine for minor things), and then click "Resolve conversation".

Reply:
Fixed